redfish-core/src/error_messages.cpp

```diff
@@ -928,6 +928,40 @@ void propertyValueIncorrect(crow::Response& res, const std::string& arg1,
     addMessageToErrorJson(res.jsonValue, propertyValueIncorrect(arg1, arg2));
 }
 
+/**
+ * @internal
+ * @brief Formats PropertyValueResourceConflict message into JSON
+ *
+ * See header file for more information
+ * @endinternal
+ */
+nlohmann::json propertyValueResourceConflict(const std::string& arg1,
+                                             const std::string& arg2,
+                                             const std::string& arg3)
```

Comment:
This is the old format, which I guess is what we want for downstream

Reply:
yes right